web/src/reducers/auth.js

```diff
@@ -8,10 +8,11 @@ import {
 } from '../actions/auth';
 
 const initialState = {
-  initialCheck: false,
-  fetching: false,
```

Comment:
A lot of this is just rearranging the properties so they're in alphabetical order. So messy otherwise. 😛